server/modelpath.go

```diff
@@ -166,3 +167,33 @@ func GetBlobsPath(digest string) (string, error) {
 
 	return path, nil
 }
+
+func migrateRegistryDomain() error {
+	manifests, err := GetManifestPath()
+	if err != nil {
+		return err
+	}
+
+	olddomainpath := filepath.Join(manifests, "registry.ollama.ai")
+	newdomainpath := filepath.Join(manifests, DefaultRegistry)
+
+	return filepath.Walk(olddomainpath, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
```

Comment:
Would this be left in a broken state if there was an error half way through for some reason? Not sure what could cause that, but I wonder if its worth trying to continue.

Reply:
It won't be broken but there'll be some models that show up as `<model>` while other models will show up as `registry.ollama.ai/library/<model>`

Reply:
Should we just log the error and continue considering that possibility?